WordPress/Classes/Services/CommentServiceRemoteCoreRESTAPI.swift

```diff
@@ -256,7 +264,7 @@ private extension CommentListParams {
             case CommentStatusFilterSpam:
                 status = .spam
             default:
-                status = nil
+                status = .custom("all")
```

Comment:
When no filter is added, fetch all comments. Otherwise, the default value "approved" will be used by the endpoint.